spot-client/src/spot-tv/ui/loaders/SpotTVRemoteControlLoader.js

```diff
@@ -39,10 +46,16 @@ export class SpotTVRemoteControlLoader extends React.Component {
 
         analytics.updateProperty('spot-tv', true);
 
-        // TODO: Add some retry logic to error handling for when the initial
-        // connection fails to be established.
         if (!this.props.isConnected && !this.props.isAttemptingConnection) {
-            this.props.dispatch(createSpotTVRemoteControlConnection());
+            if (this.props.isBackendEnabled) {
+                // TODO: persist token/pairing code and try restoring the connection
+                // Connection in the backend mode is started by entering the pairing code on the setup screen
+                history.push(ROUTES.SETUP);
```

Comment:
Maybe this could be a custom route instead? Like how there's RestrcitedRoute?

Reply:
Hmm, what would you want me to do here ? I've searched spot for `RestrictedRoute` and did not find anything.

Do you mean to create a dedicated constant and route it to the setup for now ?

Reply:
https://github.com/jitsi/jitsi-meet-spot/blob/master/spot-client/src/common/routing/RestrictedRoute.js. https://github.com/jitsi/jitsi-meet-spot/blob/master/spot-client/src/common/routing/SpotTvRestrictedRoute.js.

These make sure Spot-TV only runs on Chrome for example.